Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/CqlTranslator.java

```diff
@@ -230,7 +230,7 @@ public static String convertToJSON(Library library) throws JAXBException {
         return writer.getBuffer().toString();
     }
 
-    private static void loadModelInfo(File modelInfoXML) {
+    public static void loadModelInfo(File modelInfoXML) {
```

Comment:
Eh.  It's probably fine as is.  I just didn't understand how it fit in with the rest of this PR.